doc/source/release-notes/Percona-Server-5.7.35-38.rst

```diff
@@ -0,0 +1,31 @@
+.. _PS-5.7.35-38:
+
+================================================================================
+*Percona Server for MySQL* 5.7.35-38
+================================================================================
+
+:Date: August 18, 2021
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/5.7/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 5.7.35-38
+includes all the features and bug fixes available in
+`MySQL 5.7.35 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-35.html>`_
+in addition to enterprise-grade features developed by Percona.
```

Comment:
How about linking to these features here?
```suggestion
in addition to `enterprise-grade features <https://www.percona.com/software/mysql-database/percona-server/feature-comparison>`_ developed by Percona.
```

Reply:
Added.